tests/clock/ClockTest.cc

```diff
@@ -20,15 +20,16 @@ TEST(ClockTest, GetSimulationTimeFromClockPort)
     // Instantiate test fixture
     auto modelPath = std::filesystem::path(CMAKE_CURRENT_SOURCE_DIR) / "model.sdf";
     gz::sim::TestFixture fixture(modelPath.string());
+    fixture.Finalize();
 
-    const int iterations = 1000;
+    const int iterations = 10;
     const int deltaTns = 1e6; // 1ms
     const int tolerance = 1e6; // 1ms
     yarp::os::BufferedPort<yarp::os::Bottle> p; // Create a port.
     p.open("/tmp_in"); // Give it a name on the network.
     yarp::os::Network::connect("/clock", "/tmp_in"); // connect two ports.
-    int expectedSimTimeSeconds = (iterations - 1) / 1e3;
-    int expectedSimTimeNanoseconds = 0;
+    int expectedSimTimeSeconds = (iterations) / 1e3;
+    int expectedSimTimeNanoseconds = static_cast<int>(iterations * deltaTns) % 1000000000;
```

Comment:
```suggestion
    int expectedSimTimeNanoseconds = static_cast<int>(iterations * deltaTns) % 1e9;
```

Reply:
I would prefer to avoid this. `1e9` is a double, not an an `int`. I do not know what happens if you pass a double as second argument of `%`, but I think it is more clear and safe if we use an integere there.

Reply:
Oh yes you're right sorry. I wanted to make more understandable which number is that one. I think that from c++14 it is possible to use single quote to separate digits, hence writing `1'000'000'000`. But this is not blocking for sure.